spec/plugin.spec.js

```diff
@@ -85,7 +88,8 @@ describe('ManifestPlugin', function() {
       }, {}, function(manifest) {
         expect(manifest).toBeDefined();
         expect(manifest).toEqual({
-          'main.js': 'main.js'
+          'main.js': 'main.js',
+          'third.party.js': 'third.party.js'
```

Comment:
Can you create a new test that follow the same logic as `with ExtractTextPlugin` but for `FakeCopyWebpackPlugin`